benchmarks/collections.js

```
@@ -3,90 +3,113 @@ description: Set and Map collection benchmarks
 ---*/
 
 suite("Set operations", () => {
-  bench("add 50 elements", () => {
-    const s = new Set();
-    Array.from({ length: 50 }, (_, i) => i).forEach((i) => s.add(i));
+  bench("add 50 elements", {
+    setup: () => Array.from({ length: 50 }, (_, i) => i),
+    run: (items) => {
+      const s = new Set();
+      items.forEach((i) => s.add(i));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)